netdata-installer.sh

```diff
@@ -413,6 +421,74 @@ fi
 
 trap build_error EXIT
 
+# -----------------------------------------------------------------------------
+
+fetch_libmosquitto() {
+  download_tarball "${1}" "${2}" "libmosquitto" "mosquitto"
+}
+
+build_libmosquitto() {
+  run make -C "${1}/lib"
+}
+
+copy_libmosquitto() {
+  target_dir="${PWD}/mosquitto"
```

Comment:
This should be under an `externaldeps` directory so that it is clear that it is code that is not tracked under the repo.

Reply:
I never saw anything back from you on Slack on 2020-02-07 after Timo and I both commented that we felt it was better to leave the directory choice as-is for the moment and then update it later, and assumed that meant you either agreed or were just going to let use handle it.

Reply:
I agree to this with @Ferroin we both said let's leave it now as it is and can improve it later (due to time pressures with release coming).

Reply:
I didn’t see it on slack, but I don’t agree. I think that external dependencies need to kept separated from the code in the repo. I don’t think it is something to improve on the future as it is only the string that defines the path. Is there anything wrong with the tree that I suggested?

Reply:
@amoss I'm going to defer to @underhood on this one. It is indeed pretty trivial in the install script so I don't really care much (the whole reason I agreed with @underhood on Slack about just keeping things this way was that I had no reason to disagree, and not changing this would make development a bit faster).